### PR TITLE
Link last page in topbar

### PR DESCRIPTION
**Pull request by ad-m**

### CODE REVIEW

File: feder/cases/templates/cases/_case_small.html [hunk 1]
```diff
@@ -1,7 +1,11 @@
-{% load i18n %}
+{% load i18n humanize %}
 <h2>
     <i class="fa fa-folder-open"></i>
     <a href="{{ object.get_absolute_url }}">{{ object }}</a>
 </h2>
+<p>
```
Comment (ad-m) [inline feder/cases/templates/cases/_case_small.html:6]: Extra change for consistency with monitoring index.

File: feder/main/templates/base.html [hunk 2]
```diff
@@ -71,25 +71,26 @@ <h1 class="sr-only">{% trans 'Civil data mining' %}</h1>
...
                         <li>
+                            {# performant paginator does not support last #}
                             <a href="{% url 'cases:list' %}">
                                 <i class="fa fa-folder-open"></i>
                                 {% trans 'Cases' %}
                             </a>
                         </li>
                         <li>
-                            <a href="{% url 'monitorings:list' %}">
+                            <a href="{% url 'monitorings:list' %}?page=last">
```
Comment (ad-m) [inline feder/main/templates/base.html:81]: I don't like change order of index due jump of element over pages, but I see value to link latest page by default.

File: feder/cases/templates/cases/_case_small.html [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
+    {% trans 'Creation date' %}
```
Comment (dzejkobi) [inline feder/cases/templates/cases/_case_small.html:7]: I would put a colon between, as in following rows.

**Review by dzejkobi — COMMENTED** [review]
This does not solve all the inconveniences, because user still has to scroll down to the most recent monitoring.
But I understand the argument, that reversing results in this single view would be little inconsistent with rest of the application, so it's probably also acceptable solution.

**Pull request merged**